src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_ab2_step.jl

```diff
@@ -7,23 +7,22 @@ combine_events(free_surface_event, tracer_events) = MultiEvent(tuple(free_surfac
 
 function ab2_step!(model::HydrostaticFreeSurfaceModel, Δt, χ)
 
-    workgroup, worksize = work_layout(model.grid, :xyz)
-
     barrier = Event(device(model.architecture))
 
-    step_field_kernel! = ab2_step_field!(device(model.architecture), workgroup, worksize)
-
     # Launch velocity update kernels
 
     velocities_events = []
 
     for name in (:u, :v)
+        model.velocities isa PrescribedVelocityFields && break
```

Comment:
This function should never be called when using `PrescribedVelocityFields`. This means there is a bug in the code in `prescribed_velocity_fields.jl`. This line shouldn't be here...